datumaro/tests/test_images.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import os.path as osp
+from PIL import Image
+import timeit
+
+from unittest import TestCase
+
+from datumaro.util.test_utils import TestDir
+from datumaro.util.image import lazy_image
+
+
+class LazyImageTest(TestCase):
```

Comment:
I will recommend modify the test in accordance with the hint: https://stackoverflow.com/questions/5445080/compare-if-two-variables-reference-the-same-object-in-python